src/main/scala/org/opennetworkinsight/LDA.scala

```diff
@@ -0,0 +1,51 @@
+
```

Comment:
that is... it's grabbed from args(0) instead of coming in through a parser argument... why the difference?